zingolib/src/wallet/traits.rs

```diff
@@ -635,7 +634,7 @@ impl ReceivedNoteAndMetadata for ReceivedOrchardNoteAndMetadata {
             diversifier,
             note,
             witnessed_position,
-            nullifier: nullifier.unwrap_or(<Self::Nullifier as FromBytes<32>>::from_bytes([0; 32])),
```

Comment:
Maybe there's an anti-pattern that's suggested by this, that we should be on the lookout for.   `unwrap_or`..   maybe should be relatively rare in production code?